docs/userguide/collation/string-search.md

```diff
@@ -276,11 +276,12 @@ search service. There were some known issues in these previous releases.
 [ICU-5382](https://unicode-org.atlassian.net/browse/ICU-5382),
 [ICU-5420](https://unicode-org.atlassian.net/browse/ICU-5420))
 
-In ICU4C 4.0, the string
-search service was updated with the simple linear search algorithm, which
-locates a match by shifting a cursor in the target text one by one, and these
-issues were fixed. In ICU4C 4.0.1, the Boyer-Moore search code was reintroduced
-as a separated API set as a technology preview. In a later release, this code was deleted.
+In ICU4C 4.0, the string search service was updated to use a simple linear search
+algorithm, which locates a match by shifting a cursor in the target text one by one,
+and these issues were fixed.
+
+In ICU4C 4.0.1, the Boyer-Moore search code was reintroduced as a separate API with
+technology preview status. In ICU 51.1, this code was deleted.
```

Comment:
... Similarly, in ICU4J 53 (Java version) the Boyer-Moore search code was replaced by the simple linear search
algorithm.

?

Reply:
Ah, sorry for missing this... I'll force-push an update to change this.